hazelcast/src/main/java/com/hazelcast/util/ContextMutexFactory.java

```diff
@@ -30,30 +30,32 @@
  * <p>
  * The returned {@link Mutex}es implement {@link Closeable}, so can be conveniently used in a try-with-resources statement.
  * <p>
- * Typical usage would allow, for example, synchronizing access to a non-thread-safe {@link Map} on a per-key basis,
- * to avoid blocking other threads who would perform updates on other entries of the {@link Map}.
+ * Typical usage would allow, for example, synchronizing access to a {@link java.util.concurrent.ConcurrentMap} on a
+ * per-key basis, to avoid blocking other threads which could perform updates on other entries of the {@link Map}.
  *
- * <pre>
- *     class Test {
+ * <pre>{@code
```

Comment:
minor: `<pre>` here already indicates a pre-formatted block so `{@code}` is redundant (see http://docs.hazelcast.org/docs/3.8.6/javadoc/index.html?com/hazelcast/util/ContextMutexFactory.html ).

Reply:
I'm not sure if you're allowed to use `<` and `>` in a `<pre>` block. I guess that's why we had `&lt;` and `&gt;` there before. But it looks like you are allowed to use those literals in a `{@code} block.

Reply:
yep, exactly.

Reply:
hm, I missed this point -- the closing bracket of the `{@code }` element was missing in the original and the code sample appeared broken.